frontend/src/Components/AwardWinner/AwardWinner.jsx

```diff
@@ -34,7 +34,39 @@ function AwardWinner() {
         slidesToShow: 6,
         slidesToScroll: 1,
         nextArrow: <SampleNextArrow />,
-        prevArrow: <SamplePrevArrow />
+        prevArrow: <SamplePrevArrow />,
+        responsive: [
+          {
+              breakpoint: 1600,
+              settings: {
+                  slidesToShow: 5,
+              }
+          },
+          {
+              breakpoint: 1200,
+              settings: {
+                  slidesToShow: 4,
+              }
+          },
+          {
+              breakpoint: 900,
+              settings: {
+                  slidesToShow: 3,
+              }
+          },
+          {
+              breakpoint: 600,
+              settings: {
+                  slidesToShow: 2,
+              }
+          },
+          {
+              breakpoint: 400,
+              settings: {
+                  slidesToShow: 2,
+              }
+          }
+      ] 
       };
 
       useEffect(()=>{
```

Comment:
Please create custom hook for this and create axios service to call API 
https://dev.to/ndrohith/creating-an-axios-instance-5b4n